README.md

```diff
@@ -0,0 +1,146 @@
+Schemer
+=======
+
+![Image](app/images/schemer.gif)
+
+Schema comparison tool for Adaptive.js 1.x and Mobify.js 1.x projects.
+
+## What's a schema?
+
+A schema is a context against a view/fixture pair, with some metadata. In other
+ words, the schema will contain: 
+ 
+ - The schema name
+ - The view path the schema was generated with (relative to project folder). 
+   Not applicable for Mobify.js projects.
+ - The fixture path the schema was generated with (relative to project folder)
+ - The generated context of the view with the fixture
+
+## What does this tool do?
+
+It allows you to compare the context in the saved schema against the current
+context, and highlights any discrepancies, surfacing unintended changes to a 
+view.
+
+In other words, it lets the developer know if they've accidentally broken 
+something when changing the view.
+
+## Installation
+
+Add a reference to the Schemer repo in your project's package.json and run
+`npm install`. 
+
+You can install the current development branch in the repo by adding this:
+
+`"schemer": "git+ssh://git@github.com:mobify/schemer.git#development"`
+
+### Mobify.js Support
+
+In order to run Schemer against Mobify.js projects, a custom version of the 
+mobify-client is provided in the vendor folder. You'll also need to run 
+`npm install` in the vendor/mobify-client folder to install mobify-client 
+dependencies.
+
+## Usage
+
+In the project directory, run the following command:
+
+    ./node_modules/schemer/index.js [--interactive] [--framework=adaptivejs|mobifyjs] [--port XYZ]
+
+ **--interactive**: Run Schemer in interactive mode, accessible on 
+ http://localhost:3000/schemer.
+ 
+ **--port**: Specify the port to run schemer on
+ 
+ **--framework**: Either `adaptivejs` (default) or `mobifyjs`.
+
```

Comment:
We might want a note here about needing to have created a build for the project. So either run grunt build_dev before opening schemer, or have previewed the site at least once. 

I definitely tried using schemer on a project that I had never built and I was getting a lot of errors. Might be helpful to have some error checking within the app too.